Sources/TokamakCore/Views/Section.swift

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 7/5/20.
+//
+
+protocol SectionView {
+  func listRow(_ style: ListStyle) -> AnyView
+}
+
+public struct Section<Parent, Content, Footer> {
+  let header: Parent
+  let footer: Footer
+  let content: Content
+}
+
+extension Section: View, SectionView where Parent: View, Content: View, Footer: View {
+  public init(header: Parent, footer: Footer, @ViewBuilder content: () -> Content) {
+    self.header = header
+    self.footer = footer
+    self.content = content()
+  }
+
+  @ViewBuilder
+  public var body: TupleView<(Parent, Content, Footer)> {
```

Comment:
<!--
  1 failure:  Tuples should hav...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/Section.swift;
  Line: 36;
-->
- :no_entry_sign: Tuples should have at most 2 members. (`large_tuple`)